src/store/state/state-model.ts

```diff
@@ -30,7 +30,7 @@ export const stateModel: StateModelIF = {
     type: ''
   },
   businessInformation: {
-    legalType: '',
+    legalType: null,
```

Comment:
Needs to be null because this is now type `EntityTypes` (so empty string isn't valid).